cargo-shuttle/tests/integration/init.rs

```diff
@@ -19,7 +19,13 @@ async fn cargo_shuttle_init(path: &str) -> anyhow::Result<CommandOutcome> {
                 working_directory,
                 name: None,
             },
-            cmd: Command::Init(InitArgs { path }),
+            cmd: Command::Init(InitArgs {
+                axum: false,
+                rocket: false,
+                tide: false,
+                tower: false,
+                path,
+            }),
```

Comment:
I am happy with all the unit tests. An extra one that tests the whole sequence and that the `Cargo.toml` and `lib.rs` is correct would be nice. This can be for any framework as there is no need to test all of them in this way I think.

Maybe this test can be expanded to check the contents of `Cargo.toml` and `lib.rs`? [tempdir](docs.rs/tempdir/) can maybe be used to create the `working_directory` for this test :man_shrugging: